db/schema.rb

```diff
@@ -252,6 +252,7 @@
     t.integer "type_of_unit"
     t.string "old_id"
     t.integer "old_visible_id"
+    t.string "mobility_type"
```

Comment:
As a general rule of thumb you only want to be committing schema changes if you're also committing a migration that introduces them.